### ISSUE TITLE
Role attribute on lottie-player and accessibility

### ISSUE THREAD

**Issue by dingsoyr** [LottieFiles/lottie-player#155]
The default template for the "animation-container" has the "role" attribute hardcoded to "img".

In some cases it would be useful to be able to set the role attribute to "presentation". E.g. if the animation in question has been added to the website purly as decoration and do not contain any accessibility semantics. 

See: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/presentation_role

What is important is that if role="presentation" then the "aria-label" attribute should be removed.

**Comment by samuelOsborne** [comment]
Hi @dingsoyr thanks for suggesting this, we will look in to implementing it.

**Comment by nicolasambroise** [comment]
@theashraf, please can you also to remove the aria-label if your animation have the role="presentation".

**Comment by rcoenen** [comment]
Dev

> @dingsoyr, if you use role="presentation", it's mean that your animation is only decorative, so without controls. To be compliant with WCAG, if you have no play/pause button, you can't have the following options : loop & autoplay, and your animation should be less than 5s. (link : https://www.w3.org/WAI/WCAG21/Understanding/pause-stop-hide.html)

Devil's advocate: role=presentation refers to images according to https://www.w3.org/WAI/tutorials/images/decorative/
it's not specifically stating animations. 
So with that in mind, even the shortest, most tiny Lottie animation in the world that both auto-starts and loops should have a play/pause button. It seems completely absurd and disrupts the aesthetics to the point where you are better of not using any animations anymore... but this is possibly the outcome of WCGA 2.1 rule 2.2.2

**Comment by rcoenen** [comment]
Interesting conversation. An alternative valid approach avoiding the need for play/pause buttons on animations intended for decorative use would be using the `prefers-reduced-motion` media query to prevent decorative animations from playing altogether, but only when a user has indicated a preference for reduced motion.

From what I have been reading this is an acceptable solution to comply with WCAG 2.1 guidelines, including Success Criterion 2.2.2, which requires that users are able to control time limits for content on a webpage.

This approach respects the user's preferences and needs while allowing for the desired aesthetic to be maintained. 

Although the WCAG 2.1 Success Criterion 2.2.2 does not explicitly mention the use of the `prefers-reduced-motion` media query, it is a recommended practice to provide a more accessible experience for users who are sensitive to motion.

Thoughts?
